e/electron/electron-22.0.3-ubuntu-22.04.sh

```diff
@@ -0,0 +1,182 @@
+	# Download and extract ppc64le patches
+        wget 'https://quickbuild.io/~raptor-engineering-public/+archive/ubuntu/chromium/+files/chromium_108.0.5359.71-2raptor0~deb11u1.debian.tar.xz'
+	wget https://raw.githubusercontent.com/PF4Public/gentoo-overlay/master/dev-util/electron/files/ppc64le/fix-breakpad-compile.patch
+	wget https://raw.githubusercontent.com/PF4Public/gentoo-overlay/master/dev-util/electron/files/ppc64le/libpng-pdfium-compile-98.patch
+	wget https://raw.githubusercontent.com/PF4Public/gentoo-overlay/master/dev-util/electron/files/ppc64le/fix-swiftshader-compile.patch
+        tar -xf chromium_108.0.5359.71-2raptor0~deb11u1.debian.tar.xz
+        rm -rf chromium_108.0.5359.71-2raptor0~deb11u1.debian.tar.xz
+	
+	# Checkout source
+	mkdir -p electron && cd electron
+	gclient config --name "src/electron" --unmanaged https://github.com/electron/electron@${ELECTRON_VERSION}
+	gclient sync --with_branch_heads --with_tags -vv
+	
+	# Apply patches
+	cd src
+	sed -i "s#ppc64le/fixes/fix-breakpad-compile.patch##g" ${CWD}/debian/patches/series
+	for p in $(grep -v "^#" "${CWD}"/debian/patches/series | grep "^ppc64le"); do
+	        git apply "${CWD}/debian/patches/${p}"
+	done
+	git apply ${CWD}/fix-breakpad-compile.patch
+	git apply ${CWD}/fix-swiftshader-compile.patch
+	git apply ${CWD}/libpng-pdfium-compile-98.patch
+	sed -i 's#"-Wno-tautological-constant-out-of-range-compare"#"-Wno-tautological-constant-out-of-range-compare", "-DPNG_POWERPC_VSX_OPT=0"#g' third_party/libpng/BUILD.gn
+	sed -i "s#_CURRENT_DIR, \"..\", \"jdk\", \"current\", \"bin\", \"java\"#\"${JAVA_PATH}\", \"java\"#g" third_party/closure_compiler/compiler.py
+	sed -i 's#, "mips64"#, "mips64", "ppc64"#g' extensions/common/api/runtime.json
+	sed -i '330i \ \ } else if (strcmp(nacl_arch, "ppc64") == 0) {' chrome/browser/extensions/api/runtime/chrome_runtime_api_delegate.cc
+	sed -i '331i \ \ \ \ info->nacl_arch = extensions::api::runtime::PLATFORM_NACL_ARCH_PPC64;' chrome/browser/extensions/api/runtime/chrome_runtime_api_delegate.cc
+	sed -i '314i \ \ } else if (strcmp(arch, "ppc64") == 0) {' chrome/browser/extensions/api/runtime/chrome_runtime_api_delegate.cc
+	sed -i '315i \ \ \ \ info->arch = extensions::api::runtime::PLATFORM_ARCH_PPC64;' chrome/browser/extensions/api/runtime/chrome_runtime_api_delegate.cc
+	sed -i '84i \ \ } else if (strcmp(nacl_arch, "ppc64") == 0) {' electron/shell/browser/extensions/api/runtime/electron_runtime_api_delegate.cc
+	sed -i '85i \ \ \ \ info->nacl_arch = extensions::api::runtime::PLATFORM_NACL_ARCH_PPC64;' electron/shell/browser/extensions/api/runtime/electron_runtime_api_delegate.cc
+	sed -i '69i \ \ } else if (strcmp(arch, "ppc64") == 0) {' electron/shell/browser/extensions/api/runtime/electron_runtime_api_delegate.cc
+	sed -i '70i \ \ \ \ info->arch = extensions::api::runtime::PLATFORM_ARCH_PPC64;' electron/shell/browser/extensions/api/runtime/electron_runtime_api_delegate.cc
+	
+	# Remove intermediate files
+	rm -rf ${CWD}/debian
```

Comment:
@sumitd2 - it is advisable to use "\rm -rf" instead of just "rm -rf". This is because in case user has an alias for "rm" such as "rm -i", it will kick-in here and will make the execution annoying. If you use "\rm -rf" instead, none of the aliases, if any, would be considered.